plugin/PDFDocumentPage.qml

```diff
@@ -196,12 +198,12 @@ DocumentPage {
                 //% "Share"
                 text: qsTrId("sailfish-office-me-share")
                 onClicked: {
-                    pageStack.animatorPush("Sailfish.TransferEngine.SharePage",
-                                           {
-                                               "source": page.source,
-                                               "mimeType": page.mimeType,
-                                               "serviceFilter": ["e-mail"]
-                                           })
+                    shareAction.resources = [page.source.toString()]
```

Comment:
Seems to work, not fully sure why, but the sharing action by such only handling file paths, not file urls. But should be easy to add more explicit url support, i.e. if it starts with / or file://

Reply:
Yes this was incorrect. Fixed now.